docs/data/material/getting-started/templates/checkout/PaymentForm.tsx

```diff
@@ -1,65 +1,250 @@
+              sx={{
+                display: 'flex',
+                justifyContent: 'space-between',
+                width: '100%',
+                gap: 3,
+              }}
+            >
+              <FormGrid sx={{ flexGrow: 1 }}>
+                <FormLabel required>Card number</FormLabel>
+                <OutlinedInput
+                  id="standard-basic"
+                  autoComplete="card-number"
+                  placeholder="0000 0000 0000 0000"
+                  required
+                  value={cardNumber}
+                  onChange={handleCardNumberChange}
+                />
+              </FormGrid>
+              <FormGrid sx={{ maxWidth: '20%' }}>
+                <FormLabel required>CVV</FormLabel>
+                <OutlinedInput
+                  id="cvv"
+                  autoComplete="CVV"
+                  placeholder="123"
+                  required
+                  value={cvv}
+                  onChange={handleCvvChange}
+                />
+              </FormGrid>
+            </Box>
+            <Box sx={{ display: 'flex', justifyContent: 'space-between', gap: 3 }}>
+              <FormGrid sx={{ flexGrow: 1 }}>
+                <FormLabel required>Name</FormLabel>
+                <OutlinedInput
+                  id="card-name"
+                  autoComplete="card-name"
+                  placeholder="John Smith"
+                  required
+                />
+              </FormGrid>
+              <FormGrid sx={{ flexGrow: 1 }}>
+                <FormLabel required>Expiration date</FormLabel>
+                <OutlinedInput
+                  id="card-expiration"
+                  autoComplete="card-expiration"
+                  placeholder="MM/YY"
+                  required
+                  value={expirationDate}
+                  onChange={handleExpirationDateChange}
+                />
+              </FormGrid>
+            </Box>
+          </Box>
+        </Box>
+      )}
+      {paymentType === 'creditCard' && (
+        <FormControlLabel
+          control={<Checkbox name="saveCard" />}
+          label="Remember credit card details for next time"
+          sx={{ justifyContent: 'center' }}
+        />
+      )}
+      {paymentType === 'bankTransfer' && (
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            gap: 2,
+          }}
+        >
+          <Alert severity="warning">
+            Your order will be processed once we receive the funds.
+          </Alert>
+          <Typography variant="subtitle1">Bank account</Typography>
+          <Typography variant="body1" gutterBottom>
+            Please transfer the payment to the bank account details shown below.
+          </Typography>
+          <Box sx={{ display: 'flex', gap: 1 }}>
+            <Typography variant="body1" color="text.secondary">
+              Bank:
+            </Typography>
+            <Typography variant="body1">The Bank</Typography>
```

Comment:
```suggestion
            <Typography variant="body1" fontWeight="medium">Mastercredit</Typography>
```

A random name so we reduce the number of "bank" instances (too many)